docs/codeql/codeql-for-visual-studio-code/working-with-codeql-packs-in-visual-studio-code.rst

```diff
@@ -23,8 +29,10 @@ You can then use the CodeQL CLI to publish your pack to share with others. For m
 
 Viewing CodeQL packs and their dependencies in Visual Studio Code
 -----------------------------------------------------------------
-Whether you have used the CodeQL CLI to download a CodeQL pack that someone else has created, or created your own, you can open the ``qlpack.yml`` file in the root of a CodeQL pack directory in Visual Studio Code and view the dependencies section to see what libraries the pack depends on.
+To download a CodeQL pack that someone else has created, run the **CodeQL: Download Packs** command from the Command Palette.
```

Comment:
Having watched the gif (thanks), I wonder if we need to mention that you'll need to enter the full pack name and give an example. This may be obvious to the user but I wanted to check.

Reply:
Also - potentially, this deserves its own heading, even though it will be a very short section, to make it easier for users to find when they scan this article.

Reply:
Thanks! I've added a bit of clarification and an example in https://github.com/github/codeql/pull/7661/commits/1462565810a1d63d84b2d6fec1aea443fab138a8 ⚡ 

Haven't added a new heading just yet, since we don't expect too many users to use this command at the moment. Once we've added a way to directly run queries from a downloaded pack, we can make this more discoverable! (Currently, they'd have to find the queries in their package cache, which isn't ideal...)